src/gdcdictionary/schemas/other_clinical_attribute.yaml

```diff
@@ -1127,19 +1296,36 @@ properties:
       - Adjuvant Therapy
       - Adolescence
       - Adulthood
+      - After Chemotherapy
+      - After Study Enrollment
       - Childhood
+      - End of Consolidation Therapy
+      - End of Treatment Course
+      - End of Treatment Course 1
+      - End of Treatment Course 2
+      - First Complete Response
+      - First Treatment
       - Follow-up
       - Initial Diagnosis
       - Last Contact
       - Post Adjuvant Therapy
       - Post Hormone Therapy
+      - Post Initial Treatment
       - Post Secondary Therapy
       - Postoperative
       - Preoperative
+      - Prior to Adjuvant Therapy
+      - Prior to Chemotherapy
       - Prior to Diagnosis
+      - Prior to Procurement
+      - Prior to Study Enrollment
+      - Prior to Study Registration
       - Prior to Treatment
```

Comment:
note that the ticket listed this as an enum to add - maybe we need to edit the ticket?

Reply:
Removed from ticket,  @bullenc could you remove this also from the spreadsheet?  I removed the duplicate from this PR.

Reply:
Actually, nvm, you can just pull the dictionary into your spreadsheet when it's all merged to dev.